gridData/tests/test_dx.py

```diff
@@ -10,8 +10,9 @@
 
 from . import datafiles
 
-def test_read_dx():
-    g = Grid(datafiles.DX)
+@pytest.mark.parametrize("infile", [datafiles.DX, datafiles.DXGZ])
```

Comment:
This is good, totally happy with having the gzipped file as part of the tests. It's so tiny, it doesn't matter.

Reply:
cool!